Vagrantfile

```diff
@@ -7,7 +7,7 @@ VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.box = "mesos-demo"
-  # config.vm.box_url = "http://downloads.mesosphere.io/demo/mesos-demo.box"
+  config.vm.box_url = "http://downloads.mesosphere.io/demo/mesos-demo.box"
```

Comment:
Lets not hardcode the box URL since this might change. Its better to keep this in the documentation page, rather than over here.